adhocracy4/comments_async/api.py

```diff
@@ -192,16 +217,18 @@ def list(self, request, *args, **kwargs):
             user_has_agreed = None
             use_org_terms_of_use = True
             organisation = self.content_object.project.organisation
-            org_terms_url = reverse(
-                'organisation-terms-of-use', kwargs={
-                    'organisation_slug': organisation.slug
-                }
-            )
+            try:
+                org_terms_url = reverse(
+                    'organisation-terms-of-use', kwargs={
+                        'organisation_slug': organisation.slug
+                    }
+                )
+            except NoReverseMatch:
+                raise NotImplementedError('Add org terms of use view.')
             if hasattr(request, 'user'):
                 user = request.user
                 if user.is_authenticated:
-                    user_has_agreed = \
-                        user.has_agreed_on_org_terms(organisation)
```

Comment:
Maybe we could still use user.has_agreed_on_org_terms here? Because we already have the organisation object here and then dont have to get it again in self._user_has_agreed.

Reply:
Weeeeeell, I changed it because otherwise the tests do not work. Which is a weird reason, but I just could not find anything faking a property. And I would still like to test in a4, so we cannot really use the property that we only have in a+. So, don't know. :cry:

Reply:
Ah ok, I see! But doing all these queries to make the test work also doesnt seem right, no? Maybe we can just move the failing tests to Aplus? Or is it all of them?

Reply:
Yeah, for the poll it would be all of them. And I think, if we use `user.has_agreed_on_org_terms` in the API, we should also check that it is implemented and otherwise throw an error. Just relying on that it's there in the project also does not seem like the best way to go.